stdlib/public/Concurrency/Task.swift

```diff
@@ -618,26 +639,26 @@ public struct UnsafeCurrentTask {
     self._task = task
   }
 
-  /// Returns `Task` representing the same asynchronous context as this 'UnsafeCurrentTask'.
-  ///
-  /// Operations on `Task` (unlike `UnsafeCurrentTask`) are safe to be called
-  /// from any other task (or thread).
+  /// The current task,
+  /// represented in a way that's safe to store for later use.
   public var task: Task {
     Task(_task)
   }
 
-  /// Returns `true` if the task is cancelled, and should stop executing.
+  /// A Boolean value that indicates whether the current task was canceled.
```

Comment:
Do we really spell this as "Boolean"? The swift type is `Bool` 🤔 Or as the word I'd expect it as "boolean" 🤔

Reply:
Maybe worth documenting that once this returns true it will always return true?

Reply:
Yes, this is the correct spelling per Apple Style Guide:

> Boolean (adj.) Note capitalization.

<https://help.apple.com/applestyleguide/#/apsg1a3a0436>